mozilla/fx-private-relay
# Tips area

~~(Still set to draft because [the feature-parity PRs](https://github.com/mozilla/fx-private-relay/issues?q=is%3Aopen+label%3Areact+draft%3Afalse) have higher priority. Additionally, this still lists tips that have been moved to the New Feature Announcement area, and do not have final content yet, to be able to demo the functionality.)~~

# New feature description

Adds a "Help & Tips" area that can give the user pointers at relevant moments.

# Screenshot (if applicable)

https://user-images.githubusercontent.com/4251/159662158-af185af8-9953-4cf2-bae2-35ae1c4a7746.mp4

# How to test

Log in with a user with a subdomain set, but no custom aliases yet. You should see at least a tip explaining how to create custom aliases. The title will be shown relatively large at first. You can then either minimise it, or hit "Learn more" to expand the full tip. After minimising, it should not be shown in the initial relatively large size anymore for that user.

# Checklist
